versions.lock

```diff
@@ -87,6 +88,7 @@ io.dropwizard.metrics:metrics-jvm:4.0.2 (2 constraints: 241e5546)
 io.dropwizard.metrics:metrics-logback:4.0.2 (1 constraints: 7e0edc4f)
 io.dropwizard.metrics:metrics-servlets:4.0.2 (1 constraints: 400d371f)
 io.zipkin.java:zipkin:1.13.1 (1 constraints: 3805333b)
+javax.servlet:javax.servlet-api:3.1.0 (1 constraints: 830dcc28)
```

Comment:
I don't think this should still be here as it should be replaced by `jakarta.servlet:jakarta.servlet-api` above, right?

Reply:
this is coming from dropwizard via jetty